docs/src/main/sphinx/security/file-system-access-control.md

```diff
@@ -652,8 +652,8 @@ as `group@example.net`, you can use the following rules.
 These rules specify which users can access the system information management
 interface. System information access includes the following aspects:
 
-- Read access to details such as Trino version, uptime of the node, and others
```

Comment:
We should add that /v1/info and v1/status are always public and not affected by the rules